lib/pixel-mapper.cc

```diff
@@ -66,10 +66,57 @@ class RotatePixelMapper : public PixelMapper {
+{  12,	  13,	  14,	  15,	  28,	  29,	  30,	  31,	  44,	  45,	  46,	  47,	  60,	  61,	  62,	  63,	 140,	 141,	 142,	 143,	 156,	 157,	 158,	 159,	 172,	 173,	 174,	 175,	 188,	 189,	 190,	 191,	 268,	 269,	 270,	 271,	 284,	 285,	 286,	 287,	 300,	 301,	 302,	 303,	 316,	 317,	 318,	 319,	 396,	 397,	 398,	 399,	 412,	 413,	 414,	 415,	 428,	 429,	 430,	 431,	 444,	 445,	 446,	 447,	 524,	 525,	 526,	 527,	 540,	 541,	 542,	 543,	 556,	 557,	 558,	 559,	 572,	 573,	 574,	 575,	 652,	 653,	 654,	 655,	 668,	 669,	 670,	 671,	 684,	 685,	 686,	 687,	 700,	 701,	 702,	 703,	 780,	 781,	 782,	 783,	 796,	 797,	 798,	 799,	 812,	 813,	 814,	 815,	 828,	 829,	 830,	 831,	 908,	 909,	 910,	 911,	 924,	 925,	 926,	 927,	 940,	 941,	 942,	 943,	 956,	 957,	 958,	 959,	1036,	1037,	1038,	1039,	1052,	1053,	1054,	1055,	1068,	1069,	1070,	1071,	1084,	1085,	1086,	1087,	1164,	1165,	1166,	1167,	1180,	1181,	1182,	1183,	1196,	1197,	1198,	1199,	1212,	1213,	1214,	1215,	1292,	1293,	1294,	1295,	1308,	1309,	1310,	1311,	1324,	1325,	1326,	1327,	1340,	1341,	1342,	1343,	1420,	1421,	1422,	1423,	1436,	1437,	1438,	1439,	1452,	1453,	1454,	1455,	1468,	1469,	1470,	1471,	1548,	1549,	1550,	1551,	1564,	1565,	1566,	1567,	1580,	1581,	1582,	1583,	1596,	1597,	1598,	1599,	1676,	1677,	1678,	1679,	1692,	1693,	1694,	1695,	1708,	1709,	1710,	1711,	1724,	1725,	1726,	1727,	1804,	1805,	1806,	1807,	1820,	1821,	1822,	1823,	1836,	1837,	1838,	1839,	1852,	1853,	1854,	1855,	1932,	1933,	1934,	1935,	1948,	1949,	1950,	1951,	1964,	1965,	1966,	1967,	1980,	1981,	1982,	1983,	2060,	2061,	2062,	2063,	2076,	2077,	2078,	2079,	2092,	2093,	2094,	2095,	2108,	2109,	2110,	2111,	2188,	2189,	2190,	2191,	2204,	2205,	2206,	2207,	2220,	2221,	2222,	2223,	2236,	2237,	2238,	2239,	2316,	2317,	2318,	2319,	2332,	2333,	2334,	2335,	2348,	2349,	2350,	2351,	2364,	2365,	2366,	2367,	2444,	2445,	2446,	2447,	2460,	2461,	2462,	2463,	2476,	2477,	2478,	2479,	2492,	2493,	2494,	2495,	2572,	2573,	2574,	2575,	2588,	2589,	2590,	2591,	2604,	2605,	2606,	2607,	2620,	2621,	2622,	2623,	2700,	2701,	2702,	2703,	2716,	2717,	2718,	2719,	2732,	2733,	2734,	2735,	2748,	2749,	2750,	2751}
+    };
+
     switch (angle_) {
     case 0:
-      *matrix_x = x;
-      *matrix_y = y;
+	    *matrix_x = intLUT_xy22[y][x%intLedModuleWidth];
+		if (y <  8) 
+        *matrix_y = 0;
+      else if (y < 16)
+        *matrix_y = 1;
+      else if (y < 24)
+        *matrix_y = 2;
+      else if (y < 32)
+        *matrix_y = 3;
+      else
+        *matrix_y = 3;
```

Comment:
Restore this. Add this to lib/multiplex-mappers.c
```C++
class SingleMultiplexMapper : public MultiplexMapperBase {
public:
  SingleMultiplexMapper() : MultiplexMapperBase("Single", 8) {}

  void MapSinglePanel(int x, int y, int *matrix_x, int *matrix_y) const {
    *matrix_x = intLUT_xy22[y % 8][x % 16] * ((x / 16) + 1);
    *matrix_y = y / 8;
  }

  private:
    int intLUT_xy22[8][16] = {
      {  64,	  65,	  66,	  67,	  80,	  81,	  82,	  83,	  96,	  97,	  98,	  99,	 112,	 113,	 114,	 115 },
      {  68,	  69,	  70,	  71,	  84,	  85,	  86,	  87,	 100,	 101,	 102,	 103,	 116,	 117,	 118,	 119 },
      {  72,	  73,	  74,	  75,	  88,	  89,	  90,	  91,	 104,	 105,	 106,	 107,	 120,	 121,	 122,	 123 },
      {  76,	  77,	  78,	  79,	  92,	  93,	  94,	  95,	 108,	 109,	 110,	 111,	 124,	 125,	 126,	 127 },
      {   0,	   1,	   2,	   3,	  16,	  17,	  18,	  19,	  32,	  33,	  34,	  35,	  48,	  49,	  50,	  51 },
      {   4,	   5,	   6,	   7,	  20,	  21,	  22,	  23,	  36,	  37,	  38,	  39,	  52,	  53,	  54,	  55 },
      {   8,	   9,	  10,	  11,	  24,	  25,	  26,	  27,	  40,	  41,	  42,	  43,	  56,	  57,	  58,	  59 },
      {  12,	  13,	  14,	  15,	  28,	  29,	  30,	  31,	  44,	  45,	  46,	  47,	  60,	  61,	  62,	  63 } }
};
```

Modify this https://github.com/hzeller/rpi-rgb-led-matrix/blob/84e1465e9ea5ed000011d05369c5287eaa361ad7/lib/multiplex-mappers.cc#L467
To this
```C++
  result->push_back(new FlippedStripeMultiplexMapper());
  result->push_back(new SingleMultiplexMapper());
```